data-weave-plugin/src/main/java/org/mule/tooling/lang/dw/templates/WeaveFilesTemplateManager.java

```diff
@@ -28,8 +26,6 @@ public FileTemplateGroupDescriptor getFileTemplatesDescriptor() {
 
         //RestSDK Templates
         group.addTemplate(new FileTemplateDescriptor(CONNECTOR_DESCRIPTOR, OpenapiIcons.RepositoryLibraryLogo));
-        group.addTemplate(new FileTemplateDescriptor(RAML_API_TEMPLATE, OpenapiIcons.RepositoryLibraryLogo));
```

Comment:
Why are we deleting this feature? Users should be able to create new raml or Oas

Reply:
We discussed this with @leac87 . The idea was that that's not how the creation flow should normally go. There's always an initial API. We also discussed implementing both: importing a spec and creating a blank one.

Reply:
@machaval what do you think? Is there any concerns with this approach?

Reply:
One thing is leaving those templates so that hey can be used to create blank APIs.

The other thing is whether to support creating a blank API while creating a REST SDK project with the wizard, or just ask for an existing one to "import" as this PR implements.

I see reasons for the former, less for the latter. But I could do it... ask whether the users want to create blank API or import an existing one and adapt.

Reply:
I see now that there are templates used from the _"Create new API file"_ action and templates used by REST SDK.

**Example:**
`OAS3.yaml.ft` is used in `CreateNewApiFileAction` while `OpenApi Template.yaml.ft` is used by the REST SDK wizard. Both provide a blank API spec OAS file. I haven't touched the former.